custom_components/lampie/orchestrator.py

```diff
@@ -48,12 +55,19 @@
     from .coordinator import LampieUpdateCoordinator
 
 type ZHAEventData = dict[str, Any]
+type MQTTDeviceName = str
 
 _LOGGER = logging.getLogger(__name__)
 
+MQTT_DOMAIN: Final = "mqtt"
```

Comment:
Recommend importing these directly from the integrations constants themselves.

Reply:
👍 I'll give it a go. I tried the same with ZHA, but it would have necessitated adding an unnecessary `zha` dependency for testing. I'll try for this (and maybe also double check to see if I can import from the `const` submodule.

Reply:
Fair point - if you can import const, that should avoid the whole dependency for testing. If not, it's probably not worth it. They're unlikely to change the DOMAIN of those integrations.